instrumentation-core/spring/spring-webflux-5.0/src/main/java/io/opentelemetry/instrumentation/springwebflux/client/WebClientTracingFilter.java

```diff
@@ -34,14 +35,24 @@
 
 public class WebClientTracingFilter implements ExchangeFilterFunction {
 
+  private final Tracer tracer;
+
+  public WebClientTracingFilter() {
+    this.tracer = TRACER;
+  }
+
```

Comment:
I suggest removing the no-arg constructor and passing `TRACER` in to the constructor below